sign-csr

```diff
@@ -91,15 +91,15 @@ openssl rsa -check \
             -passin env:CA_PASS \
             -noout
 
-trap 'rm -Rf "certs/clients/$SAFE_NAME"' 0
-mkdir "certs/clients/$SAFE_NAME"
+echo -e "$NOTE Creating client directory certs/clients/$SAFE_NAME"
+mkdir -p "certs/clients/$SAFE_NAME"
```

Comment:
no need for `-p`, this directory is created at the ca initialization
https://github.com/cgzones/easy-ca/blob/sign-csr_dev/functions#L107

Reply:
mkdir will error if the dir already exits